app/templates/components/payments/funds-recipient/verification-document.hbs

```diff
@@ -0,0 +1,9 @@
+{{#if (eq stripleConnectAccount.verification_document_status 'pending_requirement')}}
+  {{!-- Show nothing --}}
+{{else if (eq stripleConnectAccount.verification_document_status 'required')}}
+  we need you to upload a scan of your ID (form + submit)
+{{else if (eq stripleConnectAccount.verification_document_status 'verifying')}}
+  Please be patient while we review the document you provided.
+{{else if (eq stripleConnectAccount.verification_document_status 'verified')}}
```

Comment:
@sarupbanskota I'm saying we only want to handle the two cases of `required` and `verifying`. No need to branch then with your conditionals. This component renders something only in those two cases.